flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/adaptive/Finished.java

```diff
@@ -54,7 +54,12 @@ public ArchivedExecutionGraph getJob() {
     }
 
     @Override
-    public void handleGlobalFailure(Throwable cause) {}
+    public void handleGlobalFailure(Throwable cause) {
+        logger.debug(
+                "Ignore global failure because we are in state {}.",
```

Comment:
nit: Why does this follow a different pattern then the other states? I expected something like 
`"Ignored global failure because we already finished the job."`

Reply:
True.